### ISSUE TITLE
the pre-trained teacher model's accuracy is 88.99 less than 92.04 in the paper

### ISSUE THREAD

**Issue by zhxjlbs** [anubhavashok/N2N#8]
CIFAR-10   resnet18    teacher model accuracy is 92.01 in the paper. I download resnet18_best.net from the link of teacher model, the accuracy is 88.99. is it the best model of resnet18 in the paper?
Thanks

**Comment by anubhavashok** [comment]
Hi are you using the same version of pytorch and the right dataset options (esp. mean subtraction)? There are two versions of the CIFAR10 model - one corresponding to dataset/cifar10.py and the other dataset/cifar10_old.py. These were tested working for v0.4.

**Comment by zhxjlbs** [comment]
Hi, I use pytorch v0.2.0, version of CIFAR10 model is dataset/cifar10.py. which one should I use? Is there a resnet18_cifar10 teacher model in the links?  Do I need to train a new teacher model?  and  when train a student model, there is a soft loss-MSELoss(teacher, student).  I have not found a hard loss-Loss(student, targets).

**Comment by zhxjlbs** [comment]
Hi, I finish the stage1 training,  Input teacher model accuracy=88.99, the best stage1 model accuracy=84.17.  
how can I get a better student model? Do you use some tricks? 
Thank you

**Comment by zhxjlbs** [comment]
> Hi are you using the same version of pytorch and the right dataset options (esp. mean subtraction)? There are two versions of the CIFAR10 model - one corresponding to dataset/cifar10.py and the other dataset/cifar10_old.py. These were tested working for v0.4.


Thank you for you advice. I use dataset/cifar10_old.py. It makes correct results.

**Comment by anubhavashok** [comment]
Great, glad to hear that!

**Comment by zhxjlbs** [comment]
> Great, glad to hear that!

I still have a little question, does the student model need to finetune?

**Comment by zhanghanqi** [comment]
does the student model need to finetune?

**Comment by zhanghanqi** [comment]
does the student model need to finetune?

**Comment by zhanghanqi** [comment]
> > Great, glad to hear that!
> 
> I still have a little question, does the student model need to finetune?

 does the student model need to finetune?
